### PR TITLE
Show multi-sig widget only to the hypha owners

### PR DESCRIPTION
**Pull request by arsenijesavic**
https://github.com/hypha-dao/dho-web-client/issues/743
https://github.com/hypha-dao/dho-web-client/issues/821

### CODE REVIEW

File: src/store/accounts/actions.js [hunk 1]
```diff
@@ -125,6 +125,17 @@ export const isAccountFree = async function (context, accountName) {
   return true
 }
 
+export const getHyphaOwners = async function ({ commit, state }) {
+  try {
+    const { account } = state
+    if (!account) return false
+    const hypha = await this.$api.getAccount('dao.hypha')
```
Comment (jmgayosso) [inline src/store/accounts/actions.js:132]: @arsenijesavic I think is better get the dao hypha name from an env variable

**Review by jmgayosso — COMMENTED** [review]
Looks good, I just did one comment regarding hypha name account, please don forget add env var too gitflow settings

**Comment by arsenijesavic** [comment]
> dao.hypha

Perfect! The env variable already exist (process.env.DAO_CONTRACT) so I will just swap

**Comment by jmgayosso** [comment]
> Ok great! amm but DAO_CONTRACT currenctly is "mtdhoxhyphaa", what exactly is dao.hypha?

Ok great! amm but DAO_CONTRACT currenctly is "mtdhoxhyphaa", what exactly is dao.hypha?

**Comment by jmgayosso** [comment]
> > dao.hypha
> 
> Perfect! The env variable already exist (process.env.DAO_CONTRACT) so I will just swap

Ok great! amm but DAO_CONTRACT currenctly is "mtdhoxhyphaa", what exactly is dao.hypha?

**Comment by arsenijesavic** [comment]
> > > dao.hypha
> > 
> > 
> > Perfect! The env variable already exist (process.env.DAO_CONTRACT) so I will just swap
> 
> Ok great! amm but DAO_CONTRACT currenctly is "mtdhoxhyphaa", what exactly is dao.hypha?

Yeah, thats test contract, production is dao.hypha 

https://github.com/hypha-dao/dho-web-client/blob/e865dc9abbcfcedd2f8787027b29c2649fa57be0/.github/workflows/deploy-prod.yml#L27

**Review by jmgayosso — APPROVED** [review]

**Pull request merged**